routes/applications.js

```diff
@@ -0,0 +1,67 @@
+const router = require("express").Router();
+const pool = require("../db_config");
+
+// Create new application
+router.post("/", (req, res) => {
+  res.send(`Here is placeholder code for adding a new job application_number`);
+});
+
+// Get single application
+router.get("/:application_number", function (req, res, next) {
+  let { application_number } = req.params;
+  const text =
+    "SELECT id, account_id, status, date_added, notes, company, position, description, salary FROM public.applications WHERE account_id = $1 AND id = $1";
+  pool.query(text, [application_number], (err, result) => {
+    if (err) {
+      console.log(err);
+      res.status(400).send(err);
+    }
+    res.status(200).send(result.rows);
+    console.log(`Application ${application_number} loaded`);
+  });
+});
+
+// Get multiple applications
+router.get("/:applications_number", function (req, res, next) { 
+  let { applications_number } = req.params;                         
+  const text =                                                 
+  "SELECT account_id, status, date_added, notes, company, position, description, salary FROM public.applications WHERE account_id = $1";
+  pool.query(text, [applications_number], (err, result) => {
```

Comment:
In line 28 you are setting account_id = $1. But here in line 29 you are passing in applications_number as the variable for $1. If you are trying to get all applications for an account you should add the Auth middleware to get the account_id and then this code would work. I did this for the account api.

Reply:
Got it thank you!